datamodel/extra_scripts/sqlserver/122_idx_gebr_doel.sql

```diff
@@ -0,0 +1,5 @@
+
+CREATE INDEX idx_gebrdoel  ON gebouwd_obj_gebruiksdoel (fk_gbo_sc_identif);
+CREATE INDEX mb_kad_onrr_z_ar_overgeg_idx  ON mb_kad_onrrnd_zk_archief (overgegaan_in);
```

Comment:
deze kan weg, deze materialized view bestaat niet in sqlserver (vervangende view heet vb_kad_onrrnd_zk_archief - maar die is waarschijnlijk niet te indexeren op deze kolom)